src/styles/layouts/_layout-about-us.scss

```diff
@@ -10,15 +13,30 @@
   height: 100vh;
 
   @include respond-above('md') {
-    grid-template-rows: map-get($header-height, 'md') 1fr map-get($footer-height, 'md');
+    grid-template-rows: map-get($header-height, 'md') 1fr map-get(
+        $footer-height,
+        'md'
+      );
   }
 
   @include respond-above('lg') {
-    grid-template-rows: map-get($header-height, 'lg') 1fr map-get($footer-height, 'lg');
+    grid-template-rows: map-get($header-height, 'lg') 1fr map-get(
+        $footer-height,
+        'lg'
+      );
   }
 
   @include respond-above('xl') {
-    grid-template-rows: map-get($header-height, 'xl') 1fr map-get($footer-height, 'xl');
+    grid-template-columns: 1fr minmax(rem(1104px), rem(1440px)) 1fr;
+    grid-template-rows: map-get($header-height, 'xl') 1fr map-get(
+        $footer-height,
+        'xl'
+      );
+    column-gap: rem(48px);
+    grid-template-areas:
+      'header header header'
+      '. main .'
+      'footer footer footer';
```

Comment:
The `width` now is correct, but you forgot to remove an extra `paddings` since there is already `column-gap`.

<img width="770" alt="Screen Shot 2564-02-24 at 15 27 43" src="https://user-images.githubusercontent.com/32289827/108985771-fe2d1600-76c3-11eb-980f-78c45f823b8b.png">

You can see that the red line is the `column-gap` but there are some `paddings` pushes them away from the gap.

<img width="327" alt="Screen Shot 2564-02-24 at 17 17 14" src="https://user-images.githubusercontent.com/32289827/108985937-316fa500-76c4-11eb-8390-423e43cfe09c.png">

In the design, it should align like this.

Reply:
```
& > main {
    grid-area: main;

    padding: rem(24px 16px);

    @include respond-above('md') {
      padding: 4.5% rem(24px);
    }

    @include respond-above('lg') {
      padding: 5% rem(32px);
    }

    @include respond-above('xl') {
      padding-left: 0;
      padding-right: 0;
    }
  }
```